src/domain/harbors/harborDetails/harborDetails.module.scss

```
@@ -1,26 +1,15 @@
 @import 'colours';
 @import 'spacings';
 
-.section {
+.column {
```

This can be simplified as follows:
```suggestion
.column {
  margin: units(1);
}
```

---

Yep, done